hoomd/hpmc/ShapeConvexPolygon.h

```diff
@@ -601,11 +601,18 @@ inline std::string getShapeSpec(const ShapeConvexPolygon& poly)
     {
     std::ostringstream shapedef;
     shapedef << "{\"type\": \"Polygon\", \"rounding_radius\": " << poly.verts.sweep_radius << ", \"vertices\": [";
-    for (unsigned int i = 0; i < poly.verts.N-1; i++)
+    if (verts.N != 0)
         {
-        shapedef << "[" << poly.verts.x[i] << ", " << poly.verts.y[i] << "], ";
+        for (unsigned int i = 0; i < poly.verts.N-1; i++)
```

Comment:
There is some inconsistency between `verts` and `poly.verts`. We probably want the shorter form? I also see below that the other diff defines `auto& verts = poly.verts;` but I don’t see that snippet here. Also that should be const.

Reply:
Good catch, I had only done the polyhedron version then decided to port to the polygon before making this PR.